benchmarks/eko/benchmark_strong_coupling.py

```diff
@@ -322,7 +322,7 @@ def benchmark_APFEL_vfns(self):
                 # print(apfel_vals_cur)
                 np.testing.assert_allclose(apfel_vals, np.array(apfel_vals_cur))
             # check myself to APFEL
-            np.testing.assert_allclose(apfel_vals, np.array(my_vals))
+            np.testing.assert_allclose(apfel_vals, np.array(my_vals), rtol=3e-3)
```

Comment:
why this is changed?

Reply:
fixed it with https://github.com/NNPDF/eko/blob/1999d92e71a8ba687f1f611af237fcd88f96e5b4/src/eko/couplings.py#L749